tests/util.py

```diff
@@ -1,9 +1,17 @@
 """Testing utilities."""
 from typing import Iterable, TypeVar
+from reasoner_util import get_preferred_ids
 
 T = TypeVar("T")
 
 
 def unordered_lists_equal(list1: Iterable[T], list2: Iterable[T]) -> bool:
     """Evaluate equality of two lists, assuming they are unordered."""
     return sorted(list1) == sorted(list2)
+
+
+def test_get_preferred_ids():
```

Comment:
This file is for utilities used for testing. Let's put the actual tests in other file(s).